File: 250228/2개의 정수를 서로 더하기/add-two-integers-each-other.py
user = input()
arr = user.split()

a = int(arr[0])
b = int(arr[1])

a += b
b += a

if 1 <= a <= 1000 and 1 <= b <= 1000 :
    print(a, b)
else :
    print("error")

File: 250721/2개의 정수를 서로 더하기/add-two-integers-each-other.swift
let input = readLine()!.split(separator: " ").map { Int(String($0))! }
let a = input[0]
let b = input[1]

if (1 <= a && a <= 1000) && (1 <= a && a <= 1000) {
    print((b + a), (b + a + b))
}
